packages/client/components/DiscussionThread.tsx

```diff
@@ -114,6 +116,8 @@ const DiscussionThread = (props: Props) => {
         ref={listRef}
         editorRef={editorRef}
         viewer={viewer}
+        showHeader={showHeader}
+        showEmptyState={showEmptyState}
```

Comment:
This was an alternative I was considering, but ultimately shied away from because of the additional effort of the refactor.  Now that you've mentioned this, too, I think it's worth doing, so I'll make this change 🙂

Reply:
awesome, looks 👌